### PR TITLE
Updated two sites in programming playground

### PR DESCRIPTION
**Pull request by harsitagarwalla187**
Added Flexbox Froggy and Flexbox Defense in Programming Playground English section of HTML and CSS.

## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?
By playing the game one can learn the use of Flex in css.
### How do we know it's really free?
I have myself played it.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a game to learn CSS flex .

### CODE REVIEW

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -142,6 +142,8 @@
 * [CodePen](https://codepen.io)
 * [CSSdeck](https://cssdeck.com)
 * [Dabblet](https://dabblet.com)
+* [Flexbox Defense](http://www.flexboxdefense.com)
+* [Flexbox Froggy](https://flexboxfroggy.com)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:146]: Already in interactive tutorials

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -409,6 +412,7 @@
 * [Bento Git Learning Track](https://bento.io/topic/git) (Bento)
 * [Bento GitHub Learning Track](https://bento.io/topic/github) (Bento)
 * [Git](https://www.youtube.com/playlist?list=PLFBirL3MAv29Vy_L7MmV2QaZLvAadFPHR) - Gwendolyn Faraday
+* [Git & Github](https://www.youtube.com/watch?v=apGV9Kg7ics) - Kunal Kushwaha(Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:415]: Use titles as it is. Alphabetize then

```suggestion
* [Complete Git and GitHub Tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics) - Kunal Kushwaha
* [Git](https://www.youtube.com/playlist?list=PLFBirL3MAv29Vy_L7MmV2QaZLvAadFPHR) - Gwendolyn Faraday
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -320,6 +320,9 @@
 * [Introduction to Databases](https://lagunita.stanford.edu/courses/Engineering/db/2014_1/about) (Stanford University)
 * [Learn SQL Basic for Data Science Specialisation](https://www.coursera.org/specializations/learn-sql-basics-data-science#about) (Coursera)
 
+### Devops
+
+* [Devops Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha(Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:325]: Fix typos

```suggestion
### DevOps

* [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
```

File: more/free-programming-playgrounds.md (hunk 1 above)
Comment (harsitagarwalla187) [reply more/free-programming-playgrounds.md:146]: Oh i didn't show that.

File: courses/free-courses-en.md (hunk 2 above)
Comment (harsitagarwalla187) [reply courses/free-courses-en.md:418]: ok

**Review by davorpa — APPROVED** [review]

**Comment by davorpa** [comment]
Sorry, Nothing to add after resolve conflicts

**Pull request closed without merging**